bin/md2gslides.js

```diff
@@ -124,9 +125,23 @@ function authorizeUser() {
     // application/utility such as this.  Of course, in such cases the "secret" is
     // actually publicly known; security depends entirely on the secrecy of refresh
     // tokens, which effectively become bearer tokens.
+
+    // Load and parse client ID and secret from client_id.json file. (Create
+    // OAuth client ID from Credentials tab at console.developers.google.com
+    // and download the credentials as client_id.json to ~/.md2googleslides
+    var data; // needs to be scoped outside of try-catch
+    try {
+        data = fs.readFileSync(STORED_CLIENT_ID_PATH);
+    } catch(err) {
+        return console.log('Error loading client secret file:', err);
+    }
+    if(data === undefined) return console.log('Error loading client secret data:', err);
+    const creds = JSON.parse(data).installed;
```

Comment:
Something that might help maintainers to get this merged is focusing on details, let's have this code more resilient. I think we have couple of things we could improve: hugs

* JSON parsing can error out easily, we should catch those! (: 
* `data == undefined` is not very helpful. Also at this point we don't have `err` variable anymore.

Also I am not the super experienced with typescript (Go dev here), is there more idiomatic way to check errors than exceptions? :thinking: 

What about something like this?

```suggestion
    var creds;
    if (!fs.existsSync(STORED_CLIENT_ID_PATH)) {
        return console.log('Client ID + Secret does not exists in path:', STORED_CLIENT_ID_PATH);
    }

    var data;
    try {
        data = fs.readFileSync(STORED_CLIENT_ID_PATH);
        creds = JSON.parse(data.toString());
    } catch (err) {
        return console.log('Error loading client secret file:', err);
    }
```

Tried here https://github.com/gsuitedevs/md2googleslides/compare/master...bwplotka:auth?expand=1 and works smoothly.

Reply:
Great! Looks good. The only thing I would change would be to shorten your warning string so line 134 is: `return console.log('Credentials file missing from:', STORED_CLIENT_ID_PATH);`, and a similar tweak can be made to line 142.